pom.xml

```diff
@@ -7,7 +7,7 @@
 
   <artifactId>kloadgen</artifactId>
 
-  <version>5.3.2</version>
+  <version>5.3.3</version>
```

Comment:
This is a good, one, but have an special POM for Maven Central Deployment. For now we have this 2 files. Hope in the future we can join in one.

Reply:
Updated pom-maven-central.xml as well